boot/bootdata/bootcd.ini

```diff
@@ -1,19 +1,33 @@
 [FREELOADER]
-DefaultOS=Setup
-TimeOut=0
+DefaultOS=LiveCD_Debug
+TimeOut=10
 
 [Display]
-TitleText=ReactOS Setup
+TitleText=ReactOS Setup & Live BootCD
```

Comment:
What information/value does the added term "Live BootCD" add here for the user? Isn't it easier to just leave that aside in the headline? And only mention those terms in the options below that headline instead?